pkg/controller/flink/job_manager_controller.go

```diff
@@ -24,7 +24,7 @@ const (
 	JobManagerPodNameFormat             = "%s-%s-jm-pod"
 	JobManagerContainerName             = "jobmanager"
 	JobManagerArg                       = "jobmanager"
-	JobManagerReadinessPath             = "/config"
+	JobManagerReadinessPath             = "/overview"
```

Comment:
Note that in the newer Flink versions, both the Jobmanagers - leader and follower respond to API requests.

I noticed that JM responds `/config` even though it is not able to process other requests.

Reply:
Looks like both the leader and follower in the latest JMs is able to respond to REST requests.

Reply:
Yeah, that's what I'm seeing as well. I think that's actually fine, the follower should redirect the request to the leader.